coral/README.md

```diff
@@ -12,9 +12,15 @@
 
 - navigate to this directory
 - run `pnpm install`
-- run `pnpm dev` to start the frontend app in development mode
+- run `pnpm dev` to start the frontend app in development mode _with remote api_
+- run `pnpm dev-without-api` to start the frontend app in development mode without api
 
-### Usage: How to run the app
+**Using a remote api**
+We recommend doing coral development with a remote API. Check out our documentation for this: [Development with remote API](docs/development-with-remote-api.md). 
+
+If you want to run Coral without an API, you can do that, too. We're using [MSW](https://mswjs.io/) to mock our API in tests. We also have setup [service-worker](src/services/api-mocks/browser.ts) that can be used to mock the API in components during development. Please do not commit this kind of mocks.  
```

Comment:
💡  very good point. That's what I get from throwing out documentation too fast :D 

Maybe it would make sense to add an example how to mock an API for development? And then specify that exactly THAT is something that shouldn't be committed :D

Reply:
Ah yes, that seems like a good idea! "I don't have a remote API to connect with but I want to have the best dev environment to work on Coral" seems like a legitimate situation to be in. And I'm sure everyone would be grateful to have examples on how to mock an API for this purpose.

Maybe it makes the scope of this PR not so small any more (but maybe not, unsure of the amount of work required ^^).

Reply:
done! :D